doc/py3-cmd.rst

```diff
@@ -39,22 +39,27 @@ You can specify the button to simulate.
 
 .. code-block:: shell
 
-    # send a click event to the whatismyip module (button 1)
```

Comment:
will we still have that default button 1 behaviour?

Reply:
Yes, sir. See `py3-cmd click`   ==> `--button INT (default 1)` (Not hardcoded).

Reply:
can we say it and show an example without --button then ?